great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -5430,7 +5432,7 @@ def save_config_variable(
             yaml.dump(config_variables, config_variables_file)
 
     def _load_fluent_config(self) -> GxConfig:
-        """Called at beginning of DataContext __init__"""
+        """Called at beginning of DataContext __init__ after config_providers init."""
```

Comment:
I wonder if we should force someone to pass in the config_providers so they know it has to happen afterwards. I haven't look into how this is implemented to see how why/how it gets the config providers.

Reply:
That's a great idea!

Reply:
done